go.mod

```diff
@@ -12,52 +12,86 @@ require (
 	github.com/fatih/color v1.16.0
 	github.com/ghodss/yaml v1.0.0
 	github.com/hashicorp/go-version v1.6.0
-	github.com/instrumenta/kubeval v0.16.1
+	github.com/instrumenta/kubeval v0.16.0
 	github.com/ivanpirog/coloredcobra v1.0.1
 	github.com/jbrukh/bayesian v0.0.0-20231117143245-13ae6f916c7a
 	github.com/manifoldco/promptui v0.9.0
 	github.com/microsoftgraph/msgraph-sdk-go v1.38.0
+	github.com/open-policy-agent/frameworks/constraint v0.0.0-20240516222118-7d1bd0255f52
+	github.com/open-policy-agent/gatekeeper/v3 v3.16.0
 	github.com/sirupsen/logrus v1.9.3
 	github.com/spf13/cobra v1.8.0
 	github.com/stretchr/testify v1.9.0
 	go.uber.org/mock v0.4.0
 	golang.org/x/exp v0.0.0-20240416160154-fe59bbe5cc7f
+	golang.org/x/mod v0.17.0
 	gopkg.in/yaml.v3 v3.0.1
 	helm.sh/helm/v3 v3.14.4
-	k8s.io/api v0.30.0
-	k8s.io/cli-runtime v0.30.0
-	k8s.io/client-go v0.30.0
+	k8s.io/api v0.29.3
```

Comment:
are we purposefully reverting to 0.29?

Reply:
unfortunately the opa contraint framework that doesn't support v0.30.0 yet, and we don't use any of the newer features so we can step down while we wait for them to update (or we can make a pr to contribute a fix to step the dependency up)  https://github.com/open-policy-agent/frameworks/blob/7d1bd0255f5233fd0a03fc28a072edbdf5f79cc7/constraint/go.mod#L16